### PR TITLE
Add animation-discharging (analog to animation-charging)

### PR DESCRIPTION
**Pull request by cs**
Would you be willing to merge this if I implement it?

### CODE REVIEW

**Comment by cs** [comment]
Heyhey, already done. This has been a lot easier than expected ;)

File: src/modules/battery.cpp [hunk 1]
```diff
@@ -327,21 +332,26 @@ namespace modules {
...
+   * use a single thread, because the two animations are never shown at the
+   * same time.
    */
   void battery_module::subthread() {
     chrono::duration<double> dur{0.0};
 
     if (m_animation_charging) {
       dur += chrono::milliseconds{m_animation_charging->framerate()};
+    } else if (m_animation_discharging) {
+      dur += chrono::milliseconds{m_animation_discharging->framerate()};
```
[the remark this answers is not in the document]
Comment (cs) [reply src/modules/battery.cpp]: Nice catch! :)

**Comment by patrick96** [comment]
Looks good, thank you very much!

**Comment by patrick96** [comment]
Added to the wiki

**Pull request merged**